Runtime/Settings/Settings.cs

```diff
@@ -41,7 +42,14 @@ protected static T Initialize()
             var filename = attribute.filename ?? typeof( T ).Name;
             var path = GetSettingsPath() + filename + ".asset";
 
-            if( attribute.usage == SettingsUsage.RuntimeProject )
+            if( attribute.usage == SettingsUsage.RuntimeUser )
+            {
+                // first load the default setting, then overwrite it
+                _instance = Resources.Load<T>( filename );
+                if (!LoadSettings())
+                    SaveSettings( true );
```

Comment:
Use a named argument here, `ignoreDirtyFlag : true`. Never have lonely bools unless they don't matter in the given context.